cmd/storage-rest-server.go

```diff
@@ -29,6 +29,7 @@ import (
 	"time"
 
 	"github.com/gorilla/mux"
+	snappy "github.com/klauspost/compress/s2"
```

Comment:
I don't like the package aliasing, since it obfuscates what it is.
```suggestion
	"github.com/klauspost/compress/s2"
```